octobot_trading/exchanges/exchange_simulator.py

```diff
@@ -14,19 +14,23 @@
 #  You should have received a copy of the GNU Lesser General Public
 #  License along with this library.
 from octobot_backtesting.api.backtesting import initialize_backtesting, modify_backtesting_timestamps, \
-    start_backtesting, get_backtesting_current_time
-from octobot_backtesting.api.importer import get_available_data_types, get_available_time_frames
+    start_backtesting, get_backtesting_current_time, set_time_updater_interval
+from octobot_backtesting.api.importer import get_available_data_types, get_available_time_frames, \
+    get_data_timestamp_interval
 from octobot_backtesting.importers.exchanges.exchange_importer import ExchangeDataImporter
+from octobot_commons.constants import MINUTE_TO_SECONDS
+from octobot_commons.enums import TimeFramesMinutes
 from octobot_commons.number_util import round_into_str_with_max_digits
 from octobot_commons.symbol_util import split_symbol
+from octobot_commons.time_frame_manager import get_config_time_frame, find_min_time_frame
 from octobot_trading.channels.exchange_channel import get_chan as get_trading_chan
 from octobot_trading.constants import CONFIG_SIMULATOR, CONFIG_DEFAULT_SIMULATOR_FEES, CONFIG_SIMULATOR_FEES, \
     CONFIG_SIMULATOR_FEES_MAKER, CONFIG_SIMULATOR_FEES_TAKER, CONFIG_SIMULATOR_FEES_WITHDRAW
 from octobot_trading.enums import ExchangeConstantsMarketStatusColumns, ExchangeConstantsMarketPropertyColumns, \
     TraderOrderType, FeePropertyColumns
 from octobot_trading.exchanges.abstract_exchange import AbstractExchange
 from octobot_trading.producers.simulator import UNAUTHENTICATED_UPDATER_SIMULATOR_PRODUCERS, \
-    SIMULATOR_PRODUCERS_TO_DATA_TYPE
+    SIMULATOR_PRODUCERS_TO_DATA_TYPE, OHLCVUpdaterSimulator
```

Comment:
I think this is not very clean to import OHLCVUpdaterSimulator here but I don't know how to fix it.

Reply:
I will change it with an enum, it will also fix imports of these in updater simulators